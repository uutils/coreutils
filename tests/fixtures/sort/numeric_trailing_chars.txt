0foo
0.abc
100 
-.05,
-x