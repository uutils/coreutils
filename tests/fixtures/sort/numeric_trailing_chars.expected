-.05,
-x
0.abc
0foo
100 
